XMonad/Layout/Minimize.hs

```diff
@@ -112,7 +112,7 @@ instance LayoutModifier Minimize Window where
                 case M.lookup w (W.floating ws) of
                   Nothing -> return $ Just $ Minimize (w:minimized) unfloated
                   Just r -> do
-                    modify (\s -> s { windowset = W.sink w ws})
+                    windows (W.sink w)
```

Comment:
What is the purpose of this change?  If I remember correctly, `windows` does not just modify the stackset, but also redraws all the windows and does some other stuff.  Is that what we want here?